docs/src/main/asciidoc/writing-extensions.adoc

```diff
@@ -2025,16 +2025,103 @@ public final class MyExtProcessor {
         resource.produce(new NativeImageResourceBuildItem("/security/runtime.keys")); //<1>
 
         resource.produce(new NativeImageResourceBuildItem(
-                "META-INF/services/" + io.quarkus.SomeService.class.getName())); //<2>
+                "META-INF/my-descriptor.xml")); //<2>
```

Comment:
why is that having an `xml` postfix?

Reply:
I didn't know what to put there, and didn't want to mention service files because that's never enough. Better suggestion?

Reply:
nevermind, I got confused from the previous example - I thought it was hinting that we have a way to load services from XML files :D

Reply:
I guess if this is about general purpose resources I'd not have it prefixed with META-INF, but I'm happy to merge it as is.